rand_core/src/block.rs

```diff
@@ -250,12 +250,12 @@ impl<R: BlockRngCore + SeedableRng> SeedableRng for BlockRng<R> {
     }
 
     #[inline(always)]
-    fn from_rng(rng: impl RngCore) -> Self {
+    fn from_rng(rng: &mut impl RngCore) -> Self {
```

Comment:
Strictly speaking, this change is not needed, but I made this signature symmetric with `try_from_rng`, which has to explicitly accept `&mut impl TryRngCore`. Plus, in our RustCrypto experience, it's easier for new users to understand `&mut impl RngCore` than to find that `impl RngCore` can accept `&mut R`.